dist/index.html

```diff
@@ -6,10 +6,95 @@
     <link rel="stylesheet" href="style.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <script src="../src/index.js"></script>
+    <script src="https://kit.fontawesome.com/9a26908d19.js"></script>
   </head>
   <body>
-    <div class="container">
-      <h1 class="center">Hello World!</h1>
+    <div class="container-parent">
+      <div class="container-header">
+        <div class="container-logo">
+            <img src="https://i.ibb.co/47tS3wH/seeklogo.png" alt="seeklogo" border="0" id="mainlogo">
+        </div>
+        <div class="container-navSection">
+          <div class="container-nav">
+                <div class="menu"> <a href="home.asp" class="fas fa-home">Home</a> </div>
+                <div class="menu"> <a href="projects.asp" class="fas fa-desktop">Projects</a> </div>
+                <div class="menu"> <a href="about.asp" class="fas fa-user">About_Me</a> </div>
+                <div class="menu"> <a href="contact.asp" class="fas fa-envelope">Contact</a> </div>
+          </div>
+          <div class="container-moto">
+            <h2>Custom Designs · Marketing · Branding · And More </h2>
+          </div>
+        </div>
+      </div>
+      <div class="container-body">
+        <div class="content">
+          <div class="content-left">
+            <span class ="ownerContact" style="color: #336D61; font-size: 3em; font-family: AlexandriaFLF;">
```

Comment:
- Get rid of the white-space between `class` and `=`
- Let’s keep a firm “separation of concerns” and keep our styling code inside `style.css`. Can you make this happen?